### PR TITLE
Partial fix for MT06383

### PR DESCRIPTION
**Pull request by dankan1890**
Exporting is now also possible for the favorites list, except if it has software in it.

### CODE REVIEW

File: src/frontend/mame/ui/selgame.cpp [hunk 1]
```diff
@@ -1355,8 +1355,21 @@ void menu_select_game::inkey_export()
...
+			// iterate over favorites
+			for (auto & favmap : mame_machine_manager::instance()->favorite().m_list)
+			{
+				if (favmap.second.startempty == 1)
+					list.push_back(favmap.second.driver);
+				else
+					return;
+			}
+		}
+		else
```
Comment (cuavas) [inline src/frontend/mame/ui/selgame.cpp]: Please indent the other line that's been made the `else` branch of this new `if` statement.  Apart from that this change is good.

**Comment by cuavas** [comment]
Line 1371 is now in the `else` branch so it should be indented by one more tab (the braces around it are fine, they don't hurt, it's just the indentation).

**Comment by dankan1890** [comment]
Okay, thanks.
For some strange reason VS displays everything correctly :(

**Comment by cuavas** [comment]
No problem, thanks.

**Pull request merged**